uml/design.uml

```diff
@@ -86,14 +113,52 @@ class Minion {
     damage_dealt
     type
     abilities
+    if_played_this_turn
 
     attack_minion(Minion)
     attack_hero(Hero)
 }
 
 class Spell {
+    Action
+
     cast()
 }
 
+class Minion_abilities {
```

Comment:
Yes, exactly. I was wondering if we should separate boolean abilities from the other ones - event triggered (like it is now - in another class Special_minion_abilities). I guess I've put only battlecry to the wrong class.